packages/edit-post/src/hooks/commands/modal-commands.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { useCommand } from '@wordpress/commands';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { cog, keyboardClose } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { KEYBOARD_SHORTCUT_HELP_MODAL_NAME } from '../../components/keyboard-shortcut-help-modal';
+import { PREFERENCES_MODAL_NAME } from '../../components/preferences-modal';
+import { store as editPostStore } from '../../store';
+
+export function useModalCommands() {
```

Comment:
Not sure about the naming here, I think this could be more generic, like `useCommonCommands` in the site editor. Not sure we need a dedicated hook for modals.